contributors/devel/sig-instrumentation/logging.md

```diff
@@ -4,14 +4,17 @@ The following conventions for the klog levels to use.
 [klog](http://godoc.org/github.com/kubernetes/klog) is globally preferred to
 [log](http://golang.org/pkg/log/) for better runtime control.
 
-Shared libraries, such as `client-go`, should not use `klog.Errorf()` and `klog.Warningf()`,
+Shared libraries, such as `client-go`, should not use `klog.ErrorS()` and `klog.InfoS()`,
 but just return `error`, because client libraries may be used in CLI UIs that wish to control output.
 
-* klog.Errorf() - Always an error
+Please see the [Structured Logging Guide](contributors/devel/sig-instrumentation/migration-to-structured-logging.md#structured-logging-in-kubernetes) for more information on how to set keys and values in structured logs.
 
-* klog.Warningf() - Something unexpected, but probably not an error
+* klog.ErrorS() - Errors should be used to indicate unexpected behaviours in code, like unexpected errors returned by subroutine function calls.
+Logs generated by `ErrorS` command may be enhanced with additional debug information (by logging library).Calling `ErrorS` with `nil` as error is semi-acceptable if there is error condition that deserves a stack trace at this origin point.
 
-* klog.Infof() has multiple levels:
+* klog.InfoS() -  Structured logs to the INFO log. Something unexpected, but probably not an error. `InfoS` should can be used for expected errors (errors that can happen during routine operations) .
```

Comment:
```suggestion
* klog.InfoS() -  Structured logs to the INFO log. `InfoS` should be used for routine logging. It can also be used to log warnings for expected errors (errors that can happen during routine operations).
```